packages/web/pages/api/bridge-transfer/quotes/[bridge].ts

```diff
@@ -39,6 +39,8 @@ export default async function quoteByBridge(
     });
   }
 
+  console.log("quoteByBridge: ", req.url);
```

Comment:
Are these console logs expected?

Reply:
No sorry, was trying to debug something. Removed now!